Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -294,10 +290,3 @@
 "captcha-reload" = "Title for button used to request another CAPTCHA to solve";
 "captcha-prompt" = "Text prompting user to solve a CAPTCHA";
 "mwk-empty-title-error" = "Error description for when an operation couldn't be performed since no title was specified.";
-"home-title" = "Title for home interface.";
```

Comment:
@bgerstle ah perhaps! 

@bgerstle @niedzielski should we maybe close this PR and try again to see if the prob goes away?